src/fosslight_dependency/package_manager/Npm.py

```diff
@@ -91,13 +94,17 @@ def parse_transitive_relationship(self):
         _version = 'version'
         _name = 'name'
 
-        cmd = 'npm ls -a --prod --json'
-        rel_tree = subprocess.check_output(cmd, shell=True, encoding='utf8')
+        cmd = 'npm ls -a --prod --json -s'
```

Comment:
And I have a question. What means the '-s' option? I cannot find this option in the [Npm cli command docs](https://docs.npmjs.com/cli/v9/commands/npm-ls).

Reply:
Silent option. Otherwise the command returns a lot more errors with the output. But yeah the npm docs are not that good,

without `-s`:
```
... object of dependencies
}
npm ERR! code ELSPROBLEMS
npm ERR! invalid: react-dom@17.0.2 /home/.../node_modules/react-dom
npm ERR! invalid: react@17.0.2 /home/.../node_modules/react
npm ERR! invalid: date-fns@1.30.1 /home/.../node_modules/date-fns
{
  "error": {
    "code": "ELSPROBLEMS",
    "summary": "invalid: react-dom@17.0.2 /home/.../node_modules/react-dom\ninvalid: react@17.0.2 /home/.../node_modules/react\ninvalid: date-fns@1.30.1 /home/.../node_modules/date-fns",
    "detail": ""
  }
}

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ruan/.npm/_logs/2023-04-17T08_03_25_100Z-debug-0.log
```

with `-s`:

```
... object of dependencies
}
{
  "error": {
    "code": "ELSPROBLEMS",
    "summary": "invalid: react-dom@17.0.2 /home/.../node_modules/react-dom\ninvalid: react@17.0.2 /home/.../node_modules/react\ninvalid: date-fns@1.30.1 /home/.../node_modules/date-fns",
    "detail": ""
  }
}
```

Reply:
Thanks for the quick explanation!